dask/dataframe/io/tests/test_sql.py

```diff
@@ -64,12 +64,17 @@ def test_divisions(db):
 
 def test_division_or_partition(db):
     with pytest.raises(TypeError):
-        read_sql_table('test', db, columns=['name'], index_col="number")
-
-    with pytest.raises(TypeError):
         read_sql_table('test', db, columns=['name'], index_col="number",
                        divisions=[0, 2, 4], npartitions=3)
 
+    import dask.dataframe.io.sql
+    dask.dataframe.io.sql.AUTO_BYTES_PER_CHUNK = 100
```

Comment:
Should probably do one of the following:

1.  Change this back to the old value using a `try-finally` block
2.  Make this a parameter of the function, so that you can provide it without mutating globals  (this might be preferred)